index.bs

```diff
@@ -10325,7 +10325,7 @@ in the node.
 				The output audio buffer that is to be consumed by the user agent. It has type <code>sequence&lt;sequence&lt;Float32Array>></code>. <code>outputs[n][m]</code> is a {{Float32Array}} object containing the audio samples for \(m\)th channel of \(n\)th output. Each of the {{Float32Array}}s are zero-filled. The number of channels in the output will match [=computedNumberOfChannels=] only when the node has a single output.
 
 			parameters:
-				A map of string keys and associated {{Float32Array}}s. <code>parameters["name"]</code> corresponds to the automation values of the {{AudioParam}} named <code>"name"</code>.
+				An <a spec="infra" lt="map">ordered map</a> of <var>name</var> → <var>parameterValues</var>. <code>parameters["name"]</code> returns <var>parameterValues</var>, which is a {{Float32Array}} with the automation values of the <var>name</var> {{AudioParam}}.
```

Comment:
Maybe `<code>parameters["<var>name</var>"</code>` so that the same styling is used for the same thing?